### ISSUE TITLE
Presenter notes font color should default to black

### ISSUE THREAD

**Issue by c-b** [gnab/remark#224]
My presentation has a black background and white font. When I add presenter notes, the font color of the notes is white, which does not show up well in the presenter screen, which uses a light grey background color. Since the presenter screen background color is hardcoded, the font color should default to black, which would allow it to be changed inline as needed (highlighting, etc). I have worked around this with:

MD:
```
???
.black[Here are the presenter notes]
```

CSS:
```
.black {
  color: #000000;
}
```

**Comment by gnab** [comment]
Good point. Fix will be included in nest release; 0.11.

**Comment by gnab** [comment]
Version 0.11 which fixed this has now been released.
